packages/x-license-pro/src/verifyLicense/verifyLicense.ts

```diff
@@ -159,7 +159,7 @@ export function verifyLicense({
   }
 
   if (!acceptedScopes.includes(license.scope)) {
-    return LicenseStatus.Invalid;
+    return LicenseStatus.OutOfScope;
```

Comment:
Maybe we should return more context on the error, here we could return the scope of the license, and the missing scope.

Reply:
@flaviendelangle Updated to be clearer.

Reply:
OK
I agree :+1:

Reply:
@oliviertassinari test case added